RUFAS/routines/manure/beddings/bedding_classes.py

```diff
@@ -11,153 +11,254 @@
-        """Return the total volume of bedding needed for all animals in the given pen.
+        """
+        Calculate the total volume of bedding needed for all animals.
 
-        Args:
-            num_animals: Number of animals in the pen.
+        Parameters
+        ----------
+        num_animals : int
+            The number of animals in the pen.
 
-        Returns:
-            Total volume of bedding needed for all animals in the given pen, m^3/day.
+        Returns
+        -------
+        float
+            The total volume of bedding needed for all animals (:math:`m^3`/day).
 
         """
         return self.calc_total_bedding_mass(num_animals) / self.bedding_density
 
     def calc_total_bedding_dry_solids(self, num_animals: int) -> float:
-        """Returns the total amount of dry solids in the bedding.
+        """
+        Calculate the total amount of dry solids in the bedding.
 
-        Args:
-            num_animals: Number of animals in the pen.
+        Parameters
+        ----------
+        num_animals : int
+            The number of animals in the pen.
 
-        Returns:
-            Total amount of dry solids in the bedding, kg/day.
+        Returns
+        -------
+        float
+            The total amount of dry solids in the bedding (kg/day).
 
         """
         return self.calc_total_bedding_mass(num_animals) / self.bedding_dry_matter_content
 
 
 class BaseOrganicBedding(BaseBedding):
-    """Base class for all organic bedding types."""
+    """
+    Abstract base class for all organic bedding types.
+
+    This class extends the BaseBedding class and provides a method to calculate the total
+    amount of bedding needed for all animals in a pen.
+
+    """
 
     def calc_total_bedding_mass(self, num_animals: int) -> float:
-        """Returns the total amount of bedding needed for all animals in the given pen.
+        """
+        Calculate the total amount of bedding needed for all animals in the given pen.
 
-        Args:
-            num_animals: Number of animals in the pen.
+        Parameters
+        ----------
+        num_animals : int
+            The number of animals in the pen.
 
-        Returns:
-            Total amount of bedding needed for all animals in the given pen, kg/day.
+        Returns
+        -------
+        float
+            Total amount of bedding needed for all animals in the given pen (kg/day).
 
         """
-        return num_animals * self.bedding_mass_per_day  # kg/day
+        return num_animals * self.bedding_mass_per_day
 
 
 class SawdustBedding(BaseOrganicBedding):
-    """Class for sawdust bedding.
+    """
+    A concrete class representing sawdust bedding.
 
-    Attributes:
-        Inherited from BaseOrganicBedding.
+    All attributes and methods are inherited from BaseOrganicBedding.
+    """
+    pass
 
+
+class CBPBSawdustBedding(BaseOrganicBedding):
+    """
+    A concrete class representing CBPB (compressed wood particles) sawdust bedding.
```

Comment:
```suggestion
    A concrete class representing compost bedded pack barn (compressed wood particles) sawdust bedding.
```
I think spelling this out as much as possible will be more readable.

Reply:
Ok